src/ui/public/sortable_column/sortable_column.less

```diff
@@ -0,0 +1,9 @@
+@import (reference) "~ui/styles/variables";
+@import (reference) "~ui/styles/mixins";
+@import (reference) "~ui/styles/theme";
+
+sortable-column {
+  .sortable-column-liner {
+    display: inline-block;
+  }
```

Comment:
Since this is being promoted in OSS kibana, this `less` should probably be replaced with css classes exposed by the UI framework.